include/clean.php

```diff
@@ -192,7 +192,7 @@ public static function cleanHtml($dirty_html, $remove_html = false)
             // remove all HTML tags
             $sugarCleaner = new SugarCleaner();
             $purifier = $sugarCleaner->purifier;
-            $clean_html = $purifier->purify($dirty_html_decoded);
+            $clean_html = $purifier->purify($dirty_html_decoded, $remove_html);
```

Comment:
As I see It waiting for a HTMLPurifier_Config instead a boolean

Reply:
I didn't recognise that the functionality has changed since 7.8. So i have added the config instead.